parm/post_avblflds.xml

```diff
@@ -6931,7 +6931,7 @@
        <post_avblfldidx>755</post_avblfldidx>
        <shortname>HWP_ON_SURFACE</shortname>
        <longname>Hourly Wildfire Potential on surface</longname>
-       <pname>FWINX</pname>
+       <pname>WFIREPOT</pname>
```

Comment:
@EricJames-NOAA May I know any reason for a new grib2 name 'WFIREPOT' replacing 'FWINX' for RRFS products?

Reply:
@WenMeng-NOAA I worked with @AndrewBenjamin-NOAA to define a new GRIB2 name for this field.  It is a new diagnostic product that we have developed at NOAA GSL, on which we are about to submit a paper to Weather and Forecasting.  I can provide more details if you are interested.

Reply:
@WenMeng-NOAA this is a new product only available for RRFS; it is not in RAP/HRRR/NAM.  Sounds good, thanks!

Reply:
@EricJames-NOAA Thanks for clarifying.